.storybook/components/CodeViewer/CodeViewer.helpers.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import reactElementToJSXString from "react-element-to-jsx-string";
+
+// modified getReactElementDisplayName from react-element-to-jsx-string
+// https://github.com/algolia/react-element-to-jsx-string/blob/master/src/parser/parseReactElement.js#L15-L20
+function getDisplayName(element) {
+  const displayName =
+    element.type.displayName ||
```

Comment:
Interesting, in my testing `element.type` is `null` if `element` is a fragment. I do see examples using fragments in the story though 🤔

Reply:
🤷‍♂ I stole this.